contrib/babelfishpg_tsql/src/rolecmds.c

```diff
@@ -585,6 +585,79 @@ grant_guests_to_login(const char *login)
 	pfree(query.data);
 }
 
+void
+grant_dbo_to_login(const char* login, const char* db_name, bool is_grant)
```

Comment:
Since we are using the same API to grant and revoke dbo, it's name should reflect the same. Lets rename it to something like grant_revoke_dbo_to_login
Please add comment for this API what actually it does.

Reply:
Done and added a comment with brief description.